src/components/messages/ThreadWindow/ThreadWindow.scss

```diff
@@ -16,6 +16,7 @@
   }
 
   &__messages {
+    height: 100%;
```

Comment:
I thought we were avoiding more `height: 100%`, why do we need this here?

Reply:
Must've been a stray I accidentally left in, thanks for pointing this out!